WebIDL/ecmascript-binding/constructors-support.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```

Comment:
If you made this a .js file you could include it in both places and thus avoid some of the duplication of these declarations inside the test file.

Reply:
Probably not worth it, because `ForeignDOMParserSubclass` needs to inherit from parent/child depending on context.